src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -888,6 +888,14 @@ public AppPage verifyHtmlMainContent(String filePath) {
         return this;
     }
     
+    public AppPage verifyHtmlAjaxMainContent(String filePath) throws Exception {
```

Comment:
It's a public method, so may be better to add some javadoc comments to this for easier reference in future.